config/module.config.php

```diff
@@ -72,9 +72,7 @@
      | AOP will check this list to apply an AOP to selected directories only,
      | leave it empty if you want AOP to be applied to all files in the appDir
      */
-    'includePaths' => [
-        $basicDirectory . '/module'
-    ],
+    'includePaths' => [],
```

Comment:
Well, no. Because `$basicDirectory . '/module'` also includes `/path/module/Application/src/Application/Aop/Aspects` for example.

And, as mentioned, the `/config/` folder as well.
The only way to use as is - is to somehow be able to exclude a wildcard path `/module/**/config`
so i don't have to deliver every config folder for every module i use. On my current project these are like 12 modules.

I could maybe include a fallback. 
Lets say the user don't add the includePath as required, we simple fall back to the default (with a notice) which then simply parses everything possible under `$basicDirectory . '/module'`